swagger.yaml

```diff
@@ -49,30 +52,51 @@ paths:
           description: New user is created
           schema:
             $ref: '#/definitions/Tutorial'
+        '405':
+          description: 'Invalid input'
+
+  /tutorials/{id}:
     delete:
-      summary: Delete tutorials with given ID
+      summary: Delete tutorial with given ID
       tags:
         - Tutorials
       responses:
-        '200':
+        '204':
           description: Tutorial is deleted
+        '400':
+          description: 'Invalid ID tutorial'
+
     patch:
       summary: Update tutorial with give ID
       tags:
         - Tutorials
       parameters:
-        - name: Tutorial
+        - name: title
```

Comment:
Why title? Please test and correct this. You can test it from swagger UI and see how it's passing data to body.